### ISSUE TITLE
ave9858.github.io: breakage

### ISSUE THREAD

**Issue by ave9858** [uBlockOrigin/uAssets#16440]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other [similarly-purposed blockers](https://twitter.com/gorhill/status/1033706103782170625).
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://ave9858.github.io/msdl/`

### Category

breakage

### Description

Using `https://ave9858.github.io/msdl/` will normally allow the generation of Windows ISO download links from microsoft, (note that heavy region and IP blocking is in place, disable VPNs), but uBlock Origin on firefox breaks the link generation due to the sessionid whitelisting request being blocked. Disabling uBlock and reloading the page fixes the issue. Example URL that is blocked but must not be: `https://vlscppe.microsoft.com/tags?org_id=y6jn8c31&session_id=00000000-0000-0000-0000-000000000000`

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://user-images.githubusercontent.com/112294121/213846016-17c1b588-e239-4c1e-bf49-f7438783cecd.png)
With uBlock Origin:

![image](https://user-images.githubusercontent.com/112294121/213846048-7e9c4047-7f56-4c9d-bded-16d6f24a6969.png)

Without:
![image](https://user-images.githubusercontent.com/112294121/213846082-3e47aaa8-4a6f-45a0-a416-d93811e8c8c7.png)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.46.0
Firefox: 109
filterset (summary): 
  network: 85189
  cosmetic: 38654
  scriptlet: 16921
  html: 868
listset (total-discarded, last-updated): 
  default: 
    user-filters: 0-0, never
    ublock-filters: 32699-55, 5m
    ublock-badware: 4364-5, 5m
    ublock-privacy: 305-0, 5m
    ublock-abuse: 77-0, 5m
    ublock-unbreak: 1905-1, 5m
    ublock-quick-fixes: 504-0, 5m
    easylist: 62162-614, 5m
    easyprivacy: 30950-1760, 5m
    urlhaus-1: 7608-0, 5m
    plowe-0: 3667-2, 5m
filterset (user): [empty]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 3236 ms
  maxAssetCacheWait: 254 ms
popupPanel: 
  blocked: 1
  blockedDetails: 
    online-metrix.net: 1
```
</details>

**Comment by ave9858** [comment]
The site was broken by commit 36e5cce1e1f0c35af1df83e403641e0e07d0d946. Although using the shared session does work, it should not be required when the current IP address is not blocked by Microsoft. Should I create a new issue? https://massgrave.dev/msdl/ uses the same code and should also be added to the unbreak list.

**Comment by stephenhawk8054** [comment]
Just continue the thread here

**Comment by stephenhawk8054** [comment]
I can download the files on that site normally. What exact is broken here?

**Comment by ave9858** [comment]
To reproduce, navigate to https://massgrave.dev/msdl/#2618 (any listed ISO works for reproducing), select a language, press confirm, and then you will get the "We are unable to complete your request at this time. Some users, entities and locations are banned from using this service." message. If you disable blocking online-metrix.net (which was done to fix this issue in bf9419688f44f0273ea9e19241b7f3a60107eaf8 but undone in 36e5cce1e1f0c35af1df83e403641e0e07d0d946), and you have an IP address not blocked by Microsoft, then you'll see the download links instead of the error. The shared session option does let you access the download links, but that option is not ideal and does not always work if too many users use it.

**Comment by stephenhawk8054** [comment]
Thanks, I can reproduce in `massgrave.dev`

**Comment by ave9858** [comment]
The issue also exists on ave9858.github.io, it's just slightly harder to notice because it automatically tries the shared session when regular link requesting fails. You can see the error message briefly appear during the loading of the download links.

**Comment by ave9858** [comment]
MSDL will be moving to `https://msdl.gravesoft.dev/`, can you add that to the exception list?
